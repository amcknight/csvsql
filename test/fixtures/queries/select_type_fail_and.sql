SELECT age AND True AS ageTrue
FROM test/fixtures/tables/people
