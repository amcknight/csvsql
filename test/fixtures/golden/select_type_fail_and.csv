Type Error in left argument of And (Var "age") (LitB True)
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
Expected Boolean but found Integer